components/Image/Image.tsx

```diff
@@ -0,0 +1,34 @@
+import NextImage, { type ImageProps } from "next/image";
+
+import styled from "styled-components";
+
+const StyledCroppedImage = styled(NextImage)`
+  position: relative;
+  overflow: hidden;
+  border-radius: 50%;
+`;
+
+const StyledImage = styled(NextImage)`
+  display: inline;
+  width: auto;
+  margin: 0;
+  padding-bottom: 0.5rem;
+  object-fit: cover;
+`;
+export interface StyledImageProps extends ImageProps {
```

Comment:
```suggestion
export interface ImageProps extends NextImageProps {
```

1. If you write it like this then the props also belong semantically more to the Image component.
2. Well done with extending the props. 
3. Also personal preference. You can think about using type instead an interface. An interface has more of an object oriented background to describing the shape of object or it is used to describe APIs or public libraries for reuse. Here is an interesting article about this topic: https://dev.to/reyronald/typescript-types-or-interfaces-for-react-component-props-1408 
But there is no wrong or good in this case.

Reply:
This is a good point to be more precise. I will consider this point for a refactoring US.